views/stats.ejs

```diff
@@ -2067,19 +2067,24 @@ const metaDescription = getMetaDescription()
           <% } %>
           </div>
         <% } %>
+
+        <% if (calculated.dungeons.essence) { %>
+          <p class="stat-sub-header">Essence</p>
```

Comment:
could you re-add the essence icon

Reply:
It might look cleaner without the icon but I would be interested to see what it looks like

Reply:
I think it's more in line with the rest of the dungeon section without it... but yeah I can try to re-add it and send u a preview...

But not before a couple days, I'll be away for new year